staging/cse/windows/azurecnifunc.tests.suites/AzureCNI.Default.OverlayDualStack.conflist

```diff
@@ -0,0 +1,52 @@
+{
+  "cniVersion": "0.3.0",
+  "name": "azure",
+  "adapterName": "",
+  "plugins": [
+    {
+      "type": "azure-vnet",
+      "mode": "bridge",
+      "bridge": "azure0",
+      "executionMode": "v4swift",
+      "capabilities": {
+        "portMappings": true,
+        "dns": true
+      },
+      "ipam": {
+        "type": "azure-cns",
+        "mode": "v4overlay"
```

Comment:
for dualstack, the ipam mode should be different - https://github.com/Azure/azure-container-networking/pull/1772/files

Reply:
fixed

Reply:
@ashvindeodhar is the conflist in the pr you linked correct? it has `"executionMode": "v4swift",` and then no IPv6 addresses for nameservers or endpoint policies. is that expected?